spec.bs

```diff
@@ -501,6 +502,16 @@ Note: In this specification we define an extension in order to allow
       SPC credentials (with the extension) are otherwise full-fledged
       WebAuthn credentials.
 
+Note: At registration time, Web Authentication requires both
+      {{PublicKeyCredentialEntity/name}} and
+      {{PublicKeyCredentialUserEntity/displayName}}, although per the
+      definition of the [=user member=], implementations are not
+      required to display either of them in subsequent authentication
+      ceremonies. Of the two, as of October 2023
+      {{PublicKeyCredentialUserEntity/displayName}} is shown more
```

Comment:
```suggestion
      {{PublicKeyCredentialUserEntity/name}} is shown more
```

Surely? `name` is shown on all authenticators I know of, whereas `displayName` is only shown on Android.